### PR TITLE
1.1.20 changelog

### PR DESCRIPTION
**Pull request by benpankow**

### CODE REVIEW

**Comment by benpankow** [comment]
Current dependencies on/for this PR:
* master
  * **PR #12506** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/12506" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/12506?utm_source=stack-comment).

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,46 @@
...
+- A new `Test sensor` / `Test schedule` button that allows you to perform a dry-run of your sensor / schedule. Check out the docs on this functionality [here](https://docs.dagster.io/master/concepts/partitions-schedules-sensors/sensors#via-dagit) for sensors and [here](https://docs.dagster.io/master/concepts/partitions-schedules-sensors/sensors#via-dagit) for schedules.
+- New `bigquery_pandas_io_manager` can store and load Pandas dataframes in BigQuery.
+- [dagster-snowflake, dagster-duckdb] SnowflakeIOManagers and DuckDBIOManagers can now default to loading inputs as a specified type if a type annotation does not exist for the input.
+- [dagster-dbt] Added the ability to use the “state:” selector
+- [dagit] Added (back) tag autocompletion in the runs filter, now with improved query performance.
+- [dagit] The Dagster libraries and their versions that were used when loading definitions can now be viewed in the actions menu for each code location.
+- [k8s] The Helm chart now supports the full kubernetes env var spec for Dagit and the Daemon. E.g.
```
Comment (sryza) [inline CHANGES.md:16]:
```suggestion
- [dagster-k8s] The Helm chart now supports the full kubernetes env var spec for Dagit and the Daemon. E.g.
```

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,46 @@
...
+
+### New
+
+- The new `@graph_asset` and `@graph_multi_asset` decorators make it more ergonomic to define graph-backed assets.
+- Dagster will auto-infer dependency relationships between single-dimensionally partitioned assets and multipartitioned assets, when the single-dimensional partitions definition is a dimension of the `MultiPartitionsDefinition`.
+- A new `Test sensor` / `Test schedule` button that allows you to perform a dry-run of your sensor / schedule. Check out the docs on this functionality [here](https://docs.dagster.io/master/concepts/partitions-schedules-sensors/sensors#via-dagit) for sensors and [here](https://docs.dagster.io/master/concepts/partitions-schedules-sensors/sensors#via-dagit) for schedules.
+- New `bigquery_pandas_io_manager` can store and load Pandas dataframes in BigQuery.
+- [dagster-snowflake, dagster-duckdb] SnowflakeIOManagers and DuckDBIOManagers can now default to loading inputs as a specified type if a type annotation does not exist for the input.
+- [dagster-dbt] Added the ability to use the “state:” selector
+- [dagit] Added (back) tag autocompletion in the runs filter, now with improved query performance.
```
Comment (sryza) [inline CHANGES.md:14]: dagit changes should go ahead of integration changes

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,46 @@
...
+- Previously, when an asset sensor did not yield run requests, it returned an empty result. This has been updated to yield a meaningful message.
+- Fix an issue with a non-partitioned asset downstream of a partitioned asset with self-dependencies causing a GQL error in dagit.
+- [dagster-snowflake-pyspark] Fixed a bug where the PySparkTypeHandler was incorrectly loading partitioned data.
+- [dagster-k8s] Fixed an issue where [run monitoring](https://docs.dagster.io/deployment/run-monitoring#run-monitoring) sometimes failed to detect that the kubernetes job for a run had stopped, leaving the run hanging.
+
+### Documentation
+
+- (experimental) Documentation for the dynamic partitions definition is now added.
+- [dagster-snowflake] The Snowflake I/O Manager reference page now includes information on working with partitioned assets.
+- Updated contributor docs to reference our new toolchain (`ruff`, `pyright`).
```
Comment (sryza) [inline CHANGES.md:41]: Should go above the dagster-snowflake entry

**Review by sryza — APPROVED** [review]

**Pull request merged**